### PR TITLE
[pcre] Add pkgconfig files

### PR DESCRIPTION
**Pull request by c72578**
**Describe the pull request**
Add **`pkgconfig`** files to **`pcre`**

- What does your PR fix? Fixes #
So far, `libpcre*.pc` files are missing. This enables `pcre` to be found using `pkgconfig`

- Which triplets are supported/not supported? Have you updated the CI baseline?
All current triplets are supported.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

File: ports/pcre/portfile.cmake [hunk 1]
```diff
@@ -50,6 +50,53 @@ foreach(FILE ${CURRENT_PACKAGES_DIR}/include/pcre.h ${CURRENT_PACKAGES_DIR}/incl
     file(WRITE ${FILE} "${PCRE_H}")
 endforeach()
 
+# Create pkgconfig files
+if(VCPKG_TARGET_IS_WINDOWS)
```
Comment (Neumann-A) [inline ports/pcre/portfile.cmake:54]: Do the other platforms already install a *.pc file? Otherwise we also need to add them for them.

Comment (c72578) [reply ports/pcre/portfile.cmake:54]: No, the *.pc files are not yet installed under vcpkg for the other platforms. I will update the PR.

**Comment by c72578** [comment]
@PhoebeHui the PR is ready for review.

**Comment by PhoebeHui** [comment]
@c72578, thanks for the PR!

Could you pelase report an issue to pcre about this changes? if pcre handle that in their offical CMakelist.txt file, then we can remove it in portfile.cmake file in future.

**Review by PhoebeHui — APPROVED** [review]

**Comment by c72578** [comment]
> 
> Could you pelase report an issue to pcre about this changes? if pcre handle that in their offical CMakelist.txt file, then we can remove it in portfile.cmake file in future.

An upstream issue including a proposed patch has been submitted upstream now:
https://bugs.exim.org/show_bug.cgi?id=2583

**Pull request merged**